apps/smart-contracts/core/test/DepositHook.test.ts

```diff
@@ -149,39 +174,52 @@ describe('=> DepositHook', () => {
       ).to.revertedWith('deposits not allowed')
     })
 
-    it('reverts if deposits not allowed and account on allow list', async () => {
-      mockAllowlist.isIncluded.returns(true)
-      expect(await mockAllowlist.isIncluded(user.address)).to.be.true
-      await depositHook.connect(deployer).setDepositsAllowed(false)
-      expect(await depositHook.depositsAllowed()).to.eq(false)
+    it('succeeds if account on allowlist', async () => {
+      mockAllowlist.isIncluded.whenCalledWith(user.address).returns(true)
+      expect(await mockAllowlist.isIncluded(user.address)).to.eq(true)
 
-      await expect(
-        depositHook.connect(vault).hook(user.address, TEST_AMOUNT_BEFORE_FEE, TEST_AMOUNT_AFTER_FEE)
-      ).to.revertedWith('deposits not allowed')
+      await depositHook
+        .connect(vault)
+        .hook(user.address, TEST_AMOUNT_BEFORE_FEE, TEST_AMOUNT_AFTER_FEE)
     })
 
-    it('reverts if account not on allow list', async () => {
-      mockAllowlist.isIncluded.returns(false)
-      expect(await mockAllowlist.isIncluded(user.address)).to.be.false
-      await depositHook.connect(deployer).setDepositsAllowed(true)
-      expect(await depositHook.depositsAllowed()).to.eq(true)
+    it('succeeds if account not on allowlist, and required score = 0', async () => {
```

Comment:
```suggestion
    it('succeeds if required score = 0', async () => {
```
don't need to say 'account not on allowlist' for this test (or any of the tests below), as that is already implied from us already testing the 'succeeds if account on allowlist' case

similarly, don't need to say 'required score > 0' for the tests below